manifests/params.pp

```diff
@@ -175,9 +175,8 @@
       $mod_dir            = "${httpd_dir}/conf.modules.d"
     } else {
       $mod_dir              = $facts['operatingsystemmajrelease'] ? {
-        '7'     => "${httpd_dir}/conf.modules.d",
-        '8'     => "${httpd_dir}/conf.modules.d",
-        default => "${httpd_dir}/conf.d",
+        '6'     => "${httpd_dir}/conf.d",
```

Comment:
Where are we today with EL5 support? Is it really dead or should we keep some code for it?

Reply:
It's no longer in the metadata.json and 5 has been EOL since November 30, 2020 per https://access.redhat.com/support/policy/updates/errata#Life_Cycle_Dates.  I can add it back in but I think 6+ is the current support list?

Reply:
I agree and I'd love to see it die, but the support policy of Puppetlabs' modules is not 100% matching metadata.json (which always confuses me). For example, there's still conditionals left in the codebase. That always makes me cautious with removing it.